src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { fetchArticles } from "../utils";
+
+export default function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchArticles()
```

Comment:
What about some loading states here + in your topics list to make sure the user has a visualisation of whats going on? 👀